src/apache/conf/ssl.conf

```diff
@@ -161,3 +161,17 @@ SSLRandomSeed connect file:/dev/urandom 512
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains;"
     </IfDefine>
 </VirtualHost>
+
+#########################
+#        Snapweb        #
+#########################
+Listen 4201
+<VirtualHost *:4201>
```

Comment:
No, you have to use 2 different URLs, one for SSL and one for without SSL. IIRC, only the port changes.